packages/sanity/playwright-ct/tests/comments/CommentInput.spec.tsx

```diff
@@ -30,7 +30,6 @@ test.describe('Comments', () => {
       await page.keyboard.press('Enter')
       await expect(page.getByTestId('comments-mentions-menu')).not.toBeVisible()
       // TODO: find a way to mock `useUser`!
-      await expect($editable).toHaveText('@Loading')
```

Comment:
❓ could we add a `data-testid` to the `TextSkeleton` component and keep this expect but instead check for visibility of that component instead?

Reply:
Yes, of course - will do that asap!